test/ibmq/test_ibmq_connector.py

```diff
@@ -145,6 +146,7 @@ def test_qx_api_version(self, qe_token, qe_url):
         version = api.api_version()
         self.assertIn('new_api', version)
 
+    @skip('function is being deprecated')
```

Comment:
Can you remove them directly instead of skipping (and maybe also `test_get_job_excludes` and the auxiliary function)? We'll consider the feature fully deprecated for the api v1 :tada: